blocks/banner/banner.css

```diff
@@ -0,0 +1,179 @@
+.banner .banner-picture {
+  position: relative;
+  height: 375px;
+  padding: 0 16px;
+  max-width: 1200px;
+  background-size: cover;
+  background-position: center;
+
+}
+
+.banner .banner-element {
+  padding-bottom: 25px;
+  background-size: cover;
+  background-position: center;
+}
+
+.banner .spotlight-link {
+  color: inherit;
+}
+
+.banner .spotlight-text .title {
+  font-size: 42px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-text .subtitle {
+  font-size: 36px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-form {
+  height: 30px;
+}
+
+.banner .spotlight-input {
+  margin: 0%;
+  height: 30px;
+  box-sizing: border-box;
+  padding: 5px 10px;
+  font-size: var(--body-font-size-xs);
+  line-height: 1.5;
+  border: 2px solid var(--dark-color);
+}
+
+.banner .spotlight-input:focus {
+  border-color: var(--input-focus-border);
+  outline: 0;
+  box-shadow: inset 0 1px 1px rgba(0 0 0 / 7.5%), 0 0 8px rgba(102 175 233 /60%);
+}
+
+.banner .spotlight-button {
+  background-color: var(--btn-bg-color-3);
+  color: var(--background-color);
+  margin-bottom: 20px;
+}
+
+.banner .spotlight-form-button {
```

Comment:
I believe the hubblehomes.com button has a border-radius of 0px;
![Uploading image.png…]()